Update outdated explanation about mounting to `/home/container` (#114)
pelican-dev/docs: docs/guides/mounts.mdx

You have to configure mounts in admin Panel in order to use them with your servers. They consist of a source pad on the node and a target path where it will be mounted in the container.

> <Admonition type="warning">
>     Mounts cannot be mounted to or inside of `/home/container` or any subdirectory of it by default, you can allow it by setting `BlockBaseDirMount` to false in the Wings configuration.
> </Admonition>
> 
> <Admonition type="warning">
>     You cannot cross-mount servers such as Server A's directory into Server B.
> </Admonition>

Reviewer: ```suggestion
<Admonition type="warning">
    By default Mounts cannot target `/home/container` or any of its subdirectory, you can allow it by setting `BlockBaseDirMount` to `false` in the Wings configuration.
    You cannot cross-mount servers such as Server A's directory into Server B.
</Admonition>
```
I think we should keep it to one admonition

Author: Didn't know it was possible, I'm new to this tool.
